EIPS/eip-2786.md

```diff
@@ -12,38 +12,81 @@ requires: 2700
 ---
 
 ## Simple Summary
+
 When an Ethereum Provider becomes connected or disconnected, it will emit a `connect`/`disconnect` event.
 
 ## Abstract
+
 The Provider is said to be “connected” when it can service RPC requests to at least one chain.
 The Provider is said to be “disconnected” when it cannot service RPC requests to any chain at all.
 When the Provider switches from a "connected" state to a "disconnected" state, it will emit a `connect` event.
 When the Provider switches from a "disconnected" state to a "connected" state, it will emit a `disconnect` event.
 
 ## Motivation
+
 When an application is hooked up to an Ethereum provider, there is value in having the application be alerted of connect/disconnect events that may occur so the application can appropriately inform the user of the situation.
 It is left up to the application to decide whether to listen in on these events, and how to handle them.
 
 ## Specification
-<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
 ### Definitions
+
 #### Connected
+
 The Provider is considered `connected` when it is able to service RPC requests to at least one chain.
+
 #### Disconnected
+
 The Provider is considered `disconnected` when it is unable to service RPC requests to any chain.
+
 ### Events
+
 #### `connect`
+
 The Provider **MUST** emit a `connect` event to all attached [EIP-2700](./eip-2700) listeners if it transitions from a `disconnected` state to a `connected` state.
-All attached listeners **MUST** be called with the parameters `('connected', { chainId })`.
+All attached listeners **MUST** be called with the parameter `{ chainId }`.
 `chainId` **MUST** specify the integer ID of the connected chain encoded as a hexadecimal string.
 If the Provider supports the `eth_chainId` JSON-RPC method or a derivation of it, then the `chainId` **MUST** match the return value of `eth_chainId`.
 The Provider **MAY** call the attached listeners in any order.
 
+#### `disconnect`
```

Comment:
This is why it is good to have someone review your work.  😬  I totally forgot to include disconnect in the connect/disconnect spec!